test/e2e/network/service.go

```diff
@@ -3456,7 +3456,7 @@ func execAffinityTestForSessionAffinityTimeout(f *framework.Framework, cs client
 			if hosts.Len() > 1 {
 				return
 			}
-			time.Sleep(time.Duration(svcSessionAffinityTimeout) * time.Second)
+			time.Sleep(time.Duration(svcSessionAffinityTimeout+1) * time.Second)
```

Comment:
is not better to change the variable directly?

Reply:
`svcSessionAffinityTimeout` is the timeout second we set to `svc.Spec.SessionAffinityConfig`, and we'd better sleep `svcSessionAffinityTimeout+1` seconds to send another request.

Reply:
ups, my fault, I've should read the whole context.
Can you add a comment explaining that so the next lazy person like me :sweat_smile:  don't ask the same question?

Reply:
Done, PTAL. 😄

Reply:
Aren't we already accounting for a delay in the session affinity timeout?

https://github.com/kubernetes/kubernetes/blob/cf6a0868be9d684ae6e227f755444db1c2fd5b9f/test/e2e/network/service.go#L3403-L3408